public/js/controllers/ui_controller.js

```diff
@@ -47,6 +47,55 @@ export class UIController {
+                <div class="d-flex justify-content-start m-1">
+                    <div class="dropdown">
+                        <button class="btn btn-brown shadow-none dropdown-toggle btn-sm mr-3" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            Columns
+                        </button>
+                        <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                            <form>
+                                <div class="form-check dropdown-item">
+                                    <input id="unique-texts" class="form-check-input" type="checkbox">
+                                    <label for="unique-texts" class="form-check-label">Unique Texts</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input id="unique-texts-labels" class="form-check-input" type="checkbox">
+                                    <label for="unique-texts-labels" class="form-check-label">Unique Texts with a label</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input id="done" class="form-check-input" type="checkbox">
+                                    <label for="done" class="form-check-label">Done</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input id="wrong-scheme-messages" class="form-check-input" type="checkbox">
+                                    <label for="wrong-scheme-messages" class="form-check-label">Wrong Scheme messages</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input id="ws" class="form-check-input" type="checkbox">
+                                    <label for="ws" class="form-check-label">WS %</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input id="not-coded-messages" class="form-check-input" type="checkbox">
+                                    <label for="not-coded-messages" class="form-check-label">Not Coded messages</label>
+                                </div>
+                                <div class="form-check dropdown-item">
+                                    <input id="NC" class="form-check-input" type="checkbox">
+                                    <label for="NC" class="form-check-label">NC %</label>
+                                </div>
+                            </form>
+                        </div>
+                    </div>
+            
+                    <div><button class="btn btn-sm btn-brown shadow-none" id="reset">View All</button></div>
```

Comment:
move this to the drop down menu

Reply:
I think this will affect users experience.The user will have to go to the dropdown menu to reset the table, if they have filtered the table by searching a given keyword. If they were to filter the table from the dropdown menu alone it will be okay that way. I have moved the button to the end to prevent the dropdown menu from shifting to the right after filtering the table.